source/exercises/mailroom-part1.rst

```diff
@@ -35,31 +37,32 @@ Write a small command-line script called ``mailroom.py``. This script should be
 * The script should prompt the user (you) to choose from a menu of 3 actions:
   "Send a Thank You," "Create a Report," or "quit."
 
-Sending a Thank You
+Send a Thank You
 -------------------
+.. I changed heading to match title of function.
 
-* If the user (you) selects 'Send a Thank You', prompt for a Full Name.
+* If the user (you) selects 'Send a Thank You,' prompt for a Full Name.
```

Comment:
comma should be after `'` like before

Reply:
There is varying opinion on the use of commas with single quotes. I am going by the Chicago Manual of Style, which calls for the comma to be inside the single quote.

Reply:
the single quotes are there to highlight the menu option selection, maybe bold or italics better?

Reply:
Single quotes are fine and they're used throughout the assignment documents I've looked at.